tests/tests_cluster_basic.yml

```diff
@@ -0,0 +1,73 @@
+- name: Minimal cluster configuration
+  hosts: all
+  vars_files: vars.yml
+  vars:
+    ha_cluster_cluster_name: "test-cluster"
+
+  tasks:
+    - block:
+        - name: Ensure pre-shared keys and TLS certificates are not present
+          file:
+            path: "{{ item }}"
+            state: absent
+          loop:
+            - /etc/corosync/authkey
+            - /etc/pacemaker/authkey
+            - /etc/cluster/fence_xvm.key
+            - /var/lib/pcsd/pcsd.key
+            - /var/lib/pcsd/pcsd.crt
+
+        - name: Run HA Cluster role
+          include_role:
+            name: linux-system-roles.ha_cluster
+
+        - meta: flush_handlers
```

Comment:
I don't think you need this since you are not using handlers

Reply:
fixed in 1c2a9bf